controller/src/main/java/org/jboss/as/controller/ProxyOperationAddressTranslator.java

```diff
@@ -48,6 +55,31 @@ public PathAddress translateAddress(PathAddress address) {
                 return translated;
             }
 
+            public PathAddress restoreAddress(PathAddress opAddress, PathAddress translatedAddress) {
+                // original is /host=foo/server=bar/subsystem=*
+                // translated is /subsystem=*
+                // we want the result to be translated back to /host=foo/server=bar/subsystem=quux
```

Comment:
I agree this isn't explicit enough, I'll expand on the comment to make it clearer.